py3status/modules/imap.py

```diff
@@ -37,12 +36,23 @@ class Py3status:
     hide_if_zero = False
     imap_server = '<IMAP_SERVER>'
     mailbox = 'INBOX'
-    new_mail_color = ''
+    msg_unavailable = 'Mail: N/A'
```

Comment:
This option is pointless for computers, but not for laptops. This allows people with laptops to customize how `imap` should look if they don't have Internet or wifi. If they don't want `Mail: N/A` to appear, they can put '', 💀 or ☠️ in.